### DISCUSSION TITLE
Thrashed Filter records is not sortable

### DISCUSSION THREAD

**Discussion by MaharshM** [filamentphp/filament#11323] [Help]
### Package

Table Builder

### Package Version

v3.0.0

### How can we help you?

Is this a bug or am i missing something? Im displaying a number of records in desc order in my table. The records that are showing on the `blank` option from the filter are sorted correctly. I'm using the `ThrashedFilter::make()` that comes out of the box with Filament to add filters for soft-deleted records. The thrashed records are listed but nor sorted correctly. Can i sort the filtered data?

**Comment by leandrocfe** [comment]
This works on the current version is `3.2.28`.
Run `composer update -W`
